### PR TITLE
Problem: duplicated encoding when passing state overrides

### PR DESCRIPTION
**Pull request by yihuang**
use json.RawMessage to avoid decoding json unnesserarily

Closes: #XXX

## Description

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

**Review by mmsqe — APPROVED** [review]

File: rpc/backend/call_tx.go [hunk 1]
```diff
@@ -372,12 +372,17 @@ func (b *Backend) DoCall(
 		return nil, errors.New("header not found")
 	}
 
+	var bzOverrides []byte
+	if overrides != nil {
```
Comment (mmsqe) [inline rpc/backend/call_tx.go:376]: seems no difference as before, not sure why change to pointer

Comment (yihuang) [reply rpc/backend/call_tx.go:376]: make it optional

Comment (yihuang) [reply rpc/backend/call_tx.go:376]: Otherwise, it's a required argument in json-rpc

Comment (mmsqe) [reply rpc/backend/call_tx.go:376]: you is right

**Pull request merged**